shard.lock

```diff
@@ -27,4 +27,3 @@ shards:
   systemd:
     git: https://github.com/84codes/systemd.cr.git
     version: 2.0.0
-
```

Comment:
Should this new line be kept? 🙂

Reply:
no :)

Reply:
Yes!

Reply:
idk why it removes it.. but it does it automatically :s